libraries/java11/Dockerfile

```diff
@@ -0,0 +1,72 @@
+# Pull base image.
+
+RUN set -eux; \
+	apt-get update; \
+	apt-get install -y --no-install-recommends \
+		bzip2 \
+		unzip \
+		wget \
+		gnupg \
+		gnupg2 \
+     software-properties-common \
```

Comment:
Looks like you mixed tabs and spaces. Can we make this a little nicer? and I would replace the `;` with `&&`